src/sentry/sentry_apps/services/app/model.py

```diff
@@ -153,5 +153,5 @@ class SentryAppInstallationFilterArgs(TypedDict, total=False):
     organization_id: int
     uuids: list[str]
     status: int
-    api_token_id: str
+    api_token_id: int
```

Comment:
This should be fine too. We have `coerce_numbers_to_str` enabled in pydantic 2 which should cover any deploy time differences.